front_end/html/edit_assignment.html

```diff
@@ -370,4 +371,27 @@ <h2>Create assignment</h2>
             timer_div.style.display = "none";
         }
     }
+
+    $('div[contenteditable]').bind('paste', function(e) {
```

Comment:
This same code is used in multiple locations. I forget where it is, but I believe we have a file in CodeBuddy that includes Javascript functions that are shared across all pages. Please find a way to move the code into a function and put it in that file and then invoke the function from here.

A good rule to follow is that if you copy and paste the same code more than twice, it should be put inside a function. Then if you have to change anything about the logic, you only have to change it in one place.